task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1524,6 +1522,10 @@ def test_handle_requests(
         next(generator)
         msg = message.model_dump_json().encode() + b"\n"
         generator.send(msg)
+
+        if isinstance(message, GetVariable):
```

Comment:
I don't like special casing this. If we need to do that, better to have a separate test

Reply:
Yeah i thought so, let me just add a param for this to check if it needs masking?